api/proof/routes.go

```diff
@@ -0,0 +1,179 @@
+package proof
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/ElrondNetwork/elrond-go/api/errors"
+	"github.com/ElrondNetwork/elrond-go/api/middleware"
+	"github.com/ElrondNetwork/elrond-go/api/shared"
+	"github.com/ElrondNetwork/elrond-go/api/wrapper"
+	"github.com/gin-gonic/gin"
+)
+
+const (
+	getProofEndpoint    = "/proof/root-hash/:roothash/address/:address"
```

Comment:
Since both routes implemented need the roothash as a parameter, might expose in a new endpoint the current roothash of the state trie? This should be implemented by using the chainHandler (blockchain.go file) implementations. Otherwise it will be very hard to use these routes.

Reply:
Done.